FeatureToggle.Application/Requests/Queries/Login/GetAuthTokenQueryHandler.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+using FeatureToggle.Application.DTOs;
+using FeatureToggle.Domain.ConfigurationModels;
+using FeatureToggle.Domain.Entity.FeatureManagementSchema;
+using MediatR;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+
+namespace FeatureToggle.Application.Requests.Queries.Login
+{
+    public class GetAuthTokenQueryHandler(UserManager<User> userManager, IOptionsMonitor<Authentication> optionsMonitor) : IRequestHandler<GetAuthTokenQuery, LoginResponseDTO>
+    {
+        private readonly UserManager<User> _userManager = userManager;  
+        private readonly IOptionsMonitor<Authentication> _optionsMonitor = optionsMonitor;
+
+        public async Task<LoginResponseDTO> Handle(GetAuthTokenQuery request, CancellationToken cancellationToken)
+        {
+            var user = await _userManager.FindByEmailAsync(request.Email);
+            if (user != null && await _userManager.CheckPasswordAsync(user, request.Password))
+            {
+                var secretKey = _optionsMonitor.CurrentValue.JWTSecret;
```

Comment:
Suggestion - Use type instead of ```var```. Consider this from next onwards.

Reply:
noted and will keep in mind